data/VT/incentives.json

```diff
@@ -141,87 +146,88 @@
       "en": "Receive up to $400 cash back on qualifying ENERGY STAR® electric clothes dryer models.",
       "es": "Reciba hasta $400 dólares de devolución en efectivo en  modelos de secadoras eléctricas de ropa ENERGY STAR® que cumplan los requisitos."
     },
-    "start_date": "2019"
+    "start_date": "2019-07-15"
   },
   {
     "id": "VT-7",
     "authority_type": "state",
-    "authority": "vt-ev",
-    "item": "heat_pump_air_conditioner_heater",
+    "authority": "vt-efficiency-vermont",
     "payment_methods": [
       "pos_rebate"
     ],
-    "program": "vt_efficiencyVtDuctedHeatPumpRebateProgram",
+    "item": "heat_pump_air_conditioner_heater",
+    "program": "vt_efficiencyVermont_efficiencyVermontDuctedHeatPumpRebate",
     "amount": {
       "type": "dollar_amount",
-      "number": 1000,
+      "number": 2000,
+      "minimum": 1000,
       "maximum": 2000
     },
     "owner_status": [
       "homeowner"
     ],
     "short_description": {
-      "en": "Work with a contractor to install your ducted heat pump and get a $1,000-$2,000 instant discount.",
+      "en": "Work with a contractor to install your ducted heat pump and get a $1,000-$2,000 instant discount, plus a bonus for income-eligible Vermonters.",
       "es": "Trabaje con un contratista para instalar su bomba de calor con ductos y obtenga un descuento instantáneo de entre $1,000 y $2,000 dólares, además de una bonificación para los residentes de Vermont que cumplan con el nivel de ingresos."
     },
-    "start_date": "2021"
+    "start_date": "2021-11-01"
   },
   {
     "id": "VT-8",
     "authority_type": "state",
-    "authority": "vt-ev",
-    "item": "heat_pump_air_conditioner_heater",
+    "authority": "vt-efficiency-vermont",
     "payment_methods": [
       "rebate"
     ],
-    "program": "vt_efficiencyVtHeatPumpRebateIncomeBonusProgram",
+    "item": "heat_pump_air_conditioner_heater",
+    "program": "vt_efficiencyVermont_efficiencyVermontHeatPumpRebateIncomeBonus",
     "amount": {
       "type": "dollar_amount",
-      "number": 200,
+      "number": 800,
+      "minimum": 200,
       "maximum": 800
     },
     "owner_status": [
       "homeowner"
     ],
-    "low_income": "default-moderate",
     "short_description": {
-      "en": "Vermonters whose household income is at or below the moderate income guidelines qualify for a $200-$800 bonus rebate on a ducted heat pump.",
+      "en": "Vermonters with low or moderate household income may qualify for a $200-$800 bonus rebate on a ducted heat pump, depending on their utility.",
       "es": "Los residentes de Vermont cuyos ingresos familiares sean iguales o inferiores al nivel de ingresos siguiente (rango moderado) ftienen derecho a un reembolso adicional de entre $200 y $800 dólares por una bomba de calor con ductos, dependiendo de la compañía de suministro de luz que utilicen."
```

Comment:
@oyamauchi are you already tracking the need to do another pass for translations on some of these?

Reply:
Yeah, I'll try to get that ticket in today